From cb08f8094664d0f961edd7a075171147815ce97c Mon Sep 17 00:00:00 2001
Message-Id: <cb08f8094664d0f961edd7a075171147815ce97c.1413837857.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Rafael Barbalho <rafael.barbalho@intel.com>
Date: Thu, 31 Jul 2014 18:06:41 +0100
Subject: [PATCH 213/312] REVERTME [VPG]: drm/i915/chv: Disable move to stolen
 on chv

The move to stolen code doesn't know about lrc and thus crashes when trying
to access the rings.

Change-Id: I299bd0e87ff6679195f34200c331e8ba624dc6b7
Signed-off-by: Rafael Barbalho <rafael.barbalho@intel.com>
---
 drivers/gpu/drm/i915/i915_gem_stolen.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/i915_gem_stolen.c b/drivers/gpu/drm/i915/i915_gem_stolen.c
index 1bb4332..94ef905 100644
--- a/drivers/gpu/drm/i915/i915_gem_stolen.c
+++ b/drivers/gpu/drm/i915/i915_gem_stolen.c
@@ -496,7 +496,7 @@ i915_gem_object_move_to_stolen(struct drm_i915_gem_object *obj)
 	u32 size = obj->base.size;
 	int ret = 0;
 
-	if (!IS_VALLEYVIEW(dev))
+	if (!IS_VALLEYVIEW(dev) || IS_CHERRYVIEW(dev))
 		return;
 
 	if (obj->stolen) {
-- 
1.7.9.5

